### PR TITLE
feat: add new block about brl-cad

### PR DESCRIPTION
**Pull request by divyanshugarg36**
This PR adds a new block to the project page which will add a block describing some brief description of BRL-CAD community

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -3,3 +3,4 @@ export { Footer } from './Footer';
 export { Layout } from './Layout';
 export { Main } from './Main';
 export { Navigation } from './Navigation';
+export { What } from './What';
```
Comment (theamanjs) [inline src/components/index.ts]: The name of this component should be more specific, `What` looks very confusing here.

**Review by theamanjs — CHANGES_REQUESTED** [review]
I strongly think we should work on the discussion of mixins in SCSS for flex and font. Because the scss code is getting lengthier with manual writing of all the properties

**Comment by theamanjs** [comment]
@divyanshugarg36 I believe every PR should have a valid and related description for a better understanding of the brief work done in that particular PR. Can you please add a description?

File: src/components/index.ts (hunk 1 above)
Comment (divyanshugarg36) [reply src/components/index.ts]: The name is now updated to Information

**Review by theamanjs — APPROVED** [review]

**Pull request merged**